source/module_cell/module_symmetry/symmetry.cpp

```diff
@@ -1787,13 +1787,54 @@ void Symmetry::set_atom_map(const Atom* atoms)
                     if (equal(diff1, 0.0) && equal(diff2, 0.0) && equal(diff3, 0.0))
                     {
                         this->isym_rotiat_[k][ia] = ja;
-
                         break;
                     }
                 }
             }
         }
     }
+    this->reset_atom_map(atoms);
+}
+
+inline bool check_full_atom_map(const std::vector<std::vector<int>>& m)
+{
+    if (m.empty()) { return false;
+}
+    for (const auto& v1 : m) { for (const auto& v2 : v1) { if (v2 == -1) { return false; } } }
+    return true;
+}
+
+void Symmetry::reset_atom_map(const Atom* atoms)
+{
+    const double eps_start = this->epsilon;
+    while (!check_full_atom_map(this->isym_rotiat_) && this->epsilon < 100 * eps_start)
```

Comment:
100 is a magic number, the updated symmetrization operation didn't pass any "test set for symmetry"

Reply:
This is a private function finally called in the public function  `analy_sys`, which has been unit-tested.
It's true that 100 has no theoretical foundation. In this only case I've encountered, 2 is enough.